### ISSUE TITLE
Import urlconf modules to proofread against.

### ISSUE THREAD

**Issue by thedrow** [mattrobenolt/proofread#3]
Allow the PROOFREAD_SUCCESS to accept a string of a urls.py module.

**Comment by mattrobenolt** [comment]
So if I pointed it at, say:  `PROOFREAD_SUCCESS = ('foo.bar.urls',)` would it be expected to extract all urls from that?

If so, that would be pretty challenging considering urls accept regular expressions. How would I make a request against `/foo/(?P<bar>.+)/(?P<baz>\d+)/?` ?

**Comment by thedrow** [comment]
You're right. There's no way to know what the parameters are unless you specify those somehow.
With DRF I can certainly find out if the View is linked to a model and the group names are fields in the model. I guess it's also possible with Django's generic class based views.
